code/modules/antagonists/werewolf/abilities/_werewolf_ability_holder.dm

```diff
@@ -400,6 +400,10 @@
 			boutput(M, "<span class='alert'>You can't use this ability when restrained!</span>")
 			return 0
 
+		if (!isturf(src.holder.owner.loc))
```

Comment:
Aren't there some werewolf abilities you'd want to be used in a container? (Like transform)

Reply:
IDK, I'm just going with how it was originally